### PR TITLE
SEO-177260-React-Redirection-Links

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @MallikaRK,
Please find and review the changes made.
Regards,
Scholar.

### CODE REVIEW

**Review by Yvone-Atieno — APPROVED** [review]
Approved

File: ej2-react/rich-text-editor/getting-started.md [hunk 1]
```diff
@@ -347,4 +347,61 @@ Specifies the items to be rendered in the quick toolbar based on the target elem
 
  {% previewsample "page.domainurl/code-snippet/rich-text-editor/basic-cs6" %}
 
-> You can refer to our [React Rich Text Editor](https://www.syncfusion.com/react-components/react-wysiwyg-rich-text-editor) feature tour page for its groundbreaking feature representations. You can also explore our [React Rich Text Editor example](https://ej2.syncfusion.com/react/demos/#/material/rich-text-editor/rich-text-editor) that shows how to render the rich text editor tools.
\ No newline at end of file
+`[Functional-component]`
```
Comment (Aishwarya-Ganesan) [inline ej2-react/rich-text-editor/getting-started.md:350]: why added all these?

**Review by Aishwarya-Ganesan — COMMENTED** [review]
Commented

File: ej2-react/rich-text-editor/getting-started.md (hunk 1 above)
Comment (Scholar-jaruha) [reply ej2-react/rich-text-editor/getting-started.md]: resolved

**Pull request closed without merging**